soc/arc/quark_se_c1000_ss/power.c

```diff
@@ -72,8 +72,10 @@ void sys_power_state_post_ops(enum power_states state)
 #if (defined(CONFIG_SYS_POWER_LOW_POWER_STATE))
 	case SYS_POWER_STATE_CPU_LPS_1:
 		/* Expire the timer as it is disabled in SS2. */
-		u32_t limit = _arc_v2_aux_reg_read(_ARC_V2_TMR0_LIMIT);
-		_arc_v2_aux_reg_write(_ARC_V2_TMR0_COUNT, limit - 1);
+		{
+			u32_t limit = _arc_v2_aux_reg_read(_ARC_V2_TMR0_LIMIT);
+			_arc_v2_aux_reg_write(_ARC_V2_TMR0_COUNT, limit - 1);
+		}
```

Comment:
Seems like this change is not related. Added by accident?

Reply:
No, this fixes an compilation issue introduced in one of my previous PRs and not detected by CI.
I hoped for fast merge :)

Reply:
I don't mind sneaking it into this PR but shoudn't it be a separate commit?

Reply:
Change removed.